handlers.c

```diff
@@ -745,17 +745,25 @@ bool handler__lregions(globals_t * vars, char **argv, unsigned argc)
     return true;
 }
 
+static bool do_search ( scan_match_type_t m, globals_t * vars, uservalue_t * pval, char ** argv, unsigned argc );
+static bool try_get_range_values ( const char * arg, uservalue_t ** pval );
+
 /* the name of the function is for history reason, now GREATERTHAN & LESSTHAN are also handled by this function */
 bool handler__decinc(globals_t * vars, char **argv, unsigned argc)
 {
     uservalue_t val;
-    scan_match_type_t m;
+    uservalue_t * pval = &val;
+    scan_match_type_t m = MATCHUNSPECIFIED;
 
     USEPARAMS();
 
     if (argc == 1)
     {
-        zero_uservalue(&val);
+        zero_uservalue(pval);
+        if (try_get_range_values(argv[0], &pval))
```

Comment:
The intention is that K&R is used. So the brace belongs to the end of the `if` line - of cause with a space in front of it.

Looks like this:
```C
if (try_get_range_values(argv[0], &pval)) {
```

The only exception are the braces used for function definition.

There are legacy parts which don't use K&R like the `if (argc == 1)` above. We are gradually fixing that once we change something at these locations.